app/controllers/users_controller.rb

```diff
@@ -14,57 +16,41 @@ def create
   end
 
   def show
-    @user = User.find(params[:id])
     @friendship = Friendship.find_by(user_id: @user.id, friend_id: current_user.id) || Friendship.find_by(user_id: current_user.id, friend_id: @user.id)
     if current_user.id == @user.id
       user_path(@user)
-    elsif current_user.friends.include?(@user) && @friendship.approved
-      user_path(@user)
-    elsif current_user.inverse_friends.include?(@user) && @friendship.approved
+    elsif current_user.all_friends.include?(@user) && @friendship.approved
       user_path(@user)
     else
       render "private"
     end
   end
 
-  def autocomplete_username
-    term = params[:q].downcase
-    @users = User.where("lower(username) LIKE '%#{term}%'")
-    respond_to do |format|
-      format.json {
-        render json: @users.map(&:username)
-      }
-    end
```

Comment:
I'm having issues with postgres and running locally too. I need to spend some time resolving ruby env issues this weekend. :person_frowning: